root/defaults/rtorrent.rc

```diff
@@ -1,27 +1,111 @@
-execute2 = {sh,-c,/usr/bin/php7 /app/rutorrent/php/initplugins.php abc &}
-execute.nothrow = rm,/run/php/.rtorrent.sock
+execute.throw = sh, -c, (cat, "/usr/bin/php7 ", "/usr/share/webapps/rutorrent/php/initplugins.php abc &")
+execute.nothrow = rm, /run/php/.rtorrent.sock
 network.scgi.open_local = /run/php/.rtorrent.sock
 schedule = socket_chmod,0,0,"execute=chmod,0660,/run/php/.rtorrent.sock"
 schedule = socket_chgrp,0,0,"execute=chgrp,abc,/run/php/.rtorrent.sock"
-log.open_file = "rtorrent", /config/log/rtorrent/rtorrent.log
+
+## Instance layout (base paths)
+method.insert = cfg.basedir,  private|const|string, (cat,"/config/rtorrent/")
+method.insert = cfg.download, private|const|string, (cat,"/downloads/")
+method.insert = cfg.logs,     private|const|string, (cat,"/config/log/rtorrent/")
+method.insert = cfg.logfile,  private|const|string, (cat,(cfg.logs),"rtorrent-",(system.time),".log")
+method.insert = cfg.session,  private|const|string, (cat,(cfg.basedir),"rtorrent_sess/")
+method.insert = cfg.watch,    private|const|string, (cat,(cfg.basedir),"watch/")
```

Comment:
In the original config, the watch directory was defined as `/downloads/watched/*.torrent`.
I think this line should be changed to match the existing paths used.

```suggestion
method.insert = cfg.watch,    private|const|string, (cat,(cfg.download),"watched/")
```